src/domains/agents/execution/execution_agent.py

```
@@ -10,6 +10,8 @@
 from dataclasses import dataclass, field
 from enum import Enum
 
+from langchain_core.tools import BaseTool
+from langchain_core.messages import HumanMessage
```

Ruff (0.12.2)

`langchain_core.messages.HumanMessage` imported but unused

Remove unused import: `langchain_core.messages.HumanMessage`

(F401)